src/Plugins/PluginDiscovery.php

```diff
@@ -46,7 +46,7 @@ public function discover()
                     try {
                         $out[] = $this->getContainer()->get(PluginInfo::class, [$dir->getPathname()]);
                     } catch (TerminusException $e) {
-                        $this->logger->debug(
+                        $this->logger->warning(
```

Comment:
I agree that `debug` is better than `warning` in the long run. However, in the short term, I think it's really important for people who are working on plugins to see feedback when they make an error, so I thought we should downgrade this to warning until -vvv is fixed.

If you think that the verbosity is a huge problem and @TeslaDethray is okay with overriding the Application object, I'm happy to help you fix up that issue. I think it's better to commit this fix quickly to avoid inconveniencing fledgling plugin writers, tho.